voice/bxml/verbs/pause.md

```diff
@@ -1,7 +1,7 @@
 {% method %}
 
 ## XML: `<Pause>`
-The Pause verb is used to delay for a set period of time.  Silence will be heard.
+The Pause verb is used to delay verb execution for a period of time. Silence will be heard.
```

Comment:
I think `Silence will be heard`, while technically correct, is an unusually clinical way of phrasing it. Perhaps:
```
The Pause verb is used to delay verb execution for a period of time, during which our leg of the call will be silent.
```
`our leg of the call` is a bit ambiguous, but I think it's better.